projects/plutus/inc/current_sense.h

```diff
@@ -19,17 +19,18 @@ typedef struct {
 } CurrentSenseCalibrationData;
 
 typedef struct {
-  LtcAdcStorage *adc_storage;
+  LtcAdcStorage adc_storage;
   CurrentSenseCalibrationData *data;
-  int32_t current_value;
-  volatile uint8_t samples;
+  CurrentSenseValue value;
+  int32_t offset;
   CurrentSenseCallback callback;
   void *context;
 } CurrentSenseStorage;
 
-// Initialize the current sense module. Requires CurrentSenseLineData to be calibrated beforehand
+// Initialize the current sense module. Requires CurrentSenseLineData to be calibrated beforehand.
```

Comment:
Use |variable| formatting. See `throttle_calibration` for an example.

Reply:
Done